jukebot/config/info.github.py

```diff
@@ -0,0 +1 @@
+TOKEN = '<DEFINED IN token generation in app page>'
```

Comment:
@ahendy0 Should we set an environment variable instead? I feel like that would be the best way to do it